### PR TITLE
Docs: release 28.0.0 review.

### PR DESCRIPTION
**Pull request by godai78**
Docs: Release 28.0.0 docs review.

Part of: cksource/ckeditor5-internal#765

### CODE REVIEW

File: packages/ckeditor5-heading/ckeditor5-metadata.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
 		{
 			"name": "Title",
 			"className": "Title",
-			"description": "Enables support for adding the title field to your document. It helps ensure that there will always be a single title field at the beginning of the document.",
+			"description": "This package enables the support for adding the title field to your document. It helps ensure that there will always be a single title field at the beginning of the document.",
```
Comment (AnnaTomanek) [inline packages/ckeditor5-heading/ckeditor5-metadata.json:6]:
```suggestion
			"description": "This package enables support for adding the title field to your document. It helps ensure that there will always be a single title field at the beginning of the document.",
```

File: packages/ckeditor5-alignment/ckeditor5-metadata.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
 			"name": "Alignment",
 			"className": "Alignment",
 			"path": "src/alignment.js",
-			"description": "Enables support for text alignment. You can use it to align your content to left, right and center or to justify it.",
+			"description": "This package enables the support for text alignment. You can use it to align your content to the left, right and center or to justify it.",
```
[the remark this answers is not in the document]
Comment (AnnaTomanek) [reply packages/ckeditor5-alignment/ckeditor5-metadata.json:7]: The problem with adding the "This package implements/enables/..." part is that it is now used in online builder. As a full sentence this is correct, but we don't have much space there and making the texts longer with an empty intro just for the sake of grammar is counter-productive. So can you revert this particular part of the fixes for the API docs? 

I'm afraid I did not notice it in the previous PRs :disappointed:  and only realized now.

**Review by AnnaTomanek — CHANGES_REQUESTED** [review]

File: packages/ckeditor5-alignment/ckeditor5-metadata.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (godai78) [reply packages/ckeditor5-alignment/ckeditor5-metadata.json:7]: It's OK.

**Comment by godai78** [comment]
@AnnaTomanek please merge this, we need it for features' output :)

**Comment by AnnaTomanek** [comment]
There is a conflict in the alignment package metadata - looks like the URL was mistakenly copied from CKFinder? Can you fix it, @godai78?

**Comment by godai78** [comment]
I fixed it before, I'll fix it again.

**Comment by godai78** [comment]
@AnnaTomanek - done, go on and merge.

File: packages/ckeditor5-essentials/ckeditor5-metadata.json [hunk 3]
```diff
@@ -3,7 +3,7 @@
 		{
 			"name": "Essentials",
 			"className": "Essentials",
-			"description": "Includes all essential editing features like clipboard, <kbd>Enter</kbd> and <kbd>Shift</kbd>+<kbd>Enter</kbd>, typing and undo support.",
+			"description": "This package includes all essential editing features like the clipboard, <kbd>Enter</kbd> and <kbd>Shift</kbd>+<kbd>Enter</kbd> keystrokes, typing and the undo support.",
```
Comment (AnnaTomanek) [inline packages/ckeditor5-essentials/ckeditor5-metadata.json:6]: Looks like the longer version is still here? Same below. Can you review?

Comment (godai78) [reply packages/ckeditor5-essentials/ckeditor5-metadata.json]: Addressed in commit.

**Review by AnnaTomanek — APPROVED** [review]

**Pull request merged**